crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -339,7 +339,14 @@ impl Client {
         Ok(())
     }
 
-    fn get_dm_room(&self, user_id: &UserId) -> Option<room::Joined> {
+    /// Get the existing DM with the given user.
+    ///
+    /// # Arguments
+    ///
+    /// * `user_id` - The unique id of the user.
```

Comment:
I don't think we should have an "Arguments" section here, it doesn't really add anything.